webui/react/src/e2e/models/pages/Workspaces.ts

```diff
@@ -0,0 +1,25 @@
+import { BasePage } from 'e2e/models/BasePage';
+import { WorkspacesList } from 'e2e/models/components/WorkspacesList';
+import { WorkspaceDetails } from 'e2e/models/components/WorkspaceDetails';
+import { ModalRootPage } from './ModalRootPage';
+import { WorkspaceCreateModal } from 'e2e/models/components/WorkspaceCreateModal';
+
+
+/**
+ * Returns a representation of an Workspaces page.
+ * @param {Page} page - The '@playwright/test' Page being used by a test
+ */
+export class Workspaces extends BasePage {
+  readonly title: string = 'Workspaces - Determined';
+  readonly url: string = 'workspaces';
+  readonly list: WorkspacesList = new WorkspacesList({ 
+    parent:this 
+  })
+  readonly projects: WorkspaceDetails = new WorkspaceDetails({ 
+    parent:this
+  })
+  readonly modalRoot: ModalRootPage = new ModalRootPage(this._page);
```

Comment:
let's move this into base page and delete `webui/react/src/e2e/models/pages/ModalRootPage.ts`

Reply:
we discussed and we able to get rid of this